tests/configuration.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from django.conf import settings
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from django_emarsys.test_helper import MockLoggingHandler
+from django_emarsys import util
+
+
+class ModelTestCase(TestCase):
+    @override_settings()
+    def test_configuration_present(self):
+        logger = logging.getLogger(util.__file__)
+        handler = MockLoggingHandler(level='WARNING')
+        logger.addHandler(handler)
+
+        # remove all EMARSYS_* settings
+        for symbol in dir(settings):
+            if symbol.startswith('EMARSYS_'):
+                delattr(settings, symbol)
+
+        util.validate_configuration()
```

Comment:
Ah, right, I neglected that.

So you'd have to call validate_configuration with
`{key: getattr(settings, name) for name in dir(settings)}` which is not the most beautiful code, but a fair price to pay for the decoupled function. I can imagine something like this:

```python
def validate_settings(settings):
    return validate_configuration({key: getattr(settings, name) for name in dir(settings)})

def validate_configuration(conf):
    ...
```

And then you test `validate_configuration`, which has the guts that should be tested.